src/GrpcProtos/capture.proto

```diff
@@ -116,15 +120,21 @@ message InternedTracepointInfo {
   TracepointInfo intern = 2;
 }
 
-message TracepointEvent {
+message TracepointEventInfo {
```

Comment:
This proto doesn't stand on its own and its name is very generic. 
Could we get rid of this proto and put its field in `InternedTracepointEvent` and `FullTracepointEvent`? 
This would be consistent with what is done with `InternedCallstackSample` and `FullCallstackSample`.

Reply:
this makes it convenient when we translate Full->Interned, there is only one field to move. But yes this is not particularly specific to Tracepoints. Also should we have CPU in CallstackStample?

Reply:
The convenience of writing one line instead of four in a couple of places at the cost of consistency and clarity is not a reason. Let me say it again:
- it's inconsistent wit callstack samples;
- it doesn't stand on its own;
- the name "info" is abused. E.g., `TracepointInfo` means something completely different.

Reply:
> Also should we have CPU in CallstackStample?

If it wasn't there I guess not for now, I guess we can always add it.

Reply:
Ah, are you suggesting to make this proto even more generic to reuse it in other protos? As soon as we have one proto that needs one field more or less we have a problem. And it just doesn't fit with the slices. I really don't see the reason for it.

Reply:
done.. it is going to be more set(get) calls but I agree it looks much better from the interface perspective.